galaxy/templates/deployment-job.yaml

```diff
@@ -29,7 +29,7 @@ spec:
           command: ['sh', '-c', 'until nc -z -w3 {{ template "galaxy-postgresql.fullname" $ }} 5432; do echo waiting for galaxy-postgres service; sleep 1; done;']
         - name: {{ .Chart.Name }}-init-mounts
           image: alpine:3.7
-          command: ['sh', '-c', 'chown -R 101:101 /galaxy/server/database && install -o 101 -g 101 /galaxy/server/config/integrated_tool_panel.xml /galaxy/server/config/editable/integrated_tool_panel.xml']
+          command: ['sh', '-c', 'chown -R 101:101 /galaxy/server/database && install -o 101 -g 101 /galaxy/server/config/integrated_tool_panel.xml /galaxy/server/config/editable/integrated_tool_panel.xml; if [ ! -f "/galaxy/server/database/config/editable_shed_tool_conf.xml" ]; then mkdir -p /galaxy/server/database/config && echo "<?xml version=\"1.0\"?>\n<toolbox tool_path=\"database/shed_tools\">\n</toolbox>" > /galaxy/server/database/config/editable_shed_tool_conf.xml; fi']
```

Comment:
One question here is whether we should support "preinstalled shed tools" from the image. If so, I think the previous method of using the galaxy image instead of the alpine image is better, because someone could build an image with the necessary tools installed, and things will just work.

Reply:
Wouldn't those go in a different `shed tool conf` file and have them not un-installable though? (I thought that the ones that we pre-load don't get added to the database and hence can't be interactively installed/uninstalled from the UI, hence no need for them to be in a separate writable file, no?)

Reply:
If the tools are installed on the image, the install database could be on the image in addition to the tools so the tools could technically be uninstalled. After a pod restarts though, the uninstall would be reverted, unless we make a persistent copy of `shed_tool_conf.xml` and the install database and have those specified in `galaxy.yml`.

Reply:
Changed back in: https://github.com/CloudVE/galaxy-kubernetes/pull/34/commits/436a3a4acb8cf9a971c36f875d9c6f1f25bd0d7e